### DISCUSSION TITLE
[go] Add support for Go 1.21

### DISCUSSION THREAD

**Discussion by jalavosus** [vercel/community#4173] [Feature Question] [answered]
Go 1.21 is the latest version of Go. Given the [Go backwards compatibility promise](https://go.dev/doc/go1compat), I don't believe that bumping the supported version from 1.20 to 1.21 would be a big change from a library perspective (I'm basing this opinion off of the [1.20 bump PR](https://github.com/vercel/vercel/pull/9367/files)) but would be a welcome change for Go developers using Vercel as a whole. 

All of the above to say, I could be wrong, and it could be a massive issue that I'm not completely in the loop about. 

Thanks!

**Comment by TooTallNate** [comment, marked as answer]
We actually just [merged a PR for this](https://github.com/vercel/vercel/pull/10552) yesterday. It should be live at some point today.

**Reply by jalavosus** [reply]
Oh heck yeah, thanks a bunch!
